core/license.go

```diff
@@ -36,6 +36,12 @@ func SetupLicenseAndProjectHash(endpoints *cloud.QdApiEndpoints, token string) {
 				log.Fatal(err)
 			}
 		}
+		if licenseData.OrganisationIdHash != "" {
```

Comment:
What happens if in response from server there is no `organizationIdHash`? We already released older on-prem for some customers

Reply:
Where is a test in this commit specificaly for this case.